package.json

```diff
@@ -39,7 +43,8 @@
     "build": "npm run build:ts",
     "watch:ts": "tsc -p tsconfig.build.json --watch",
     "watch": "npm run watch:ts",
-    "lint": "eslint --ext .ts src"
+    "lint": "eslint --ext .ts src",
+    "test": "mocha build/test --exit"
```

Comment:
No need to compile test files. You can run TypeScript tests:
https://github.com/ioBroker/create-adapter/blob/master/.mocharc.json
https://github.com/ioBroker/create-adapter/blob/master/package.json#L66-L67
https://github.com/ioBroker/create-adapter/blob/master/package.json#L97